broker-client/src/main/java/org/aktin/broker/client2/BrokerAdmin2.java

```diff
@@ -310,6 +312,42 @@ public ResponseWithMetadata getResult(int requestId, int nodeId) throws IOExcept
 		HttpResponse<InputStream> resp = getResult(requestId, nodeId, BodyHandlers.ofInputStream());
 		return wrapResource(resp, requestId+"_result_"+nodeId);
 	}
+    
+    public String getAggregatedResultUUID(int requestId) throws IOException {
+        HttpRequest req = createBrokerRequest("export/request-bundle/" + requestId)
+                .POST(BodyPublishers.noBody()).build();
+        HttpResponse<String> response = sendRequest(req, BodyHandlers.ofString());
+        if (response.statusCode() != 200)
+            throw new IOException("Unexpected response code " + response.statusCode() + " instead of expected 200");
+        return response.body();
+    }
+    
+    public <T> HttpResponse<T> getAggregatedResult(String uuid, BodyHandler<T> handler) throws IOException {
+        HttpRequest req = createBrokerRequest("download/" + uuid).GET().build();
+        return sendRequest(req, handler);
+    }
+    
+    @Override
+    public ResponseWithMetadata getAggregatedResult(String uuid) throws IOException {
```

Comment:
Gibt es einen Grund warum das in zwei Calls aufgeteilt sein muss? (get..ID und get..Result) Aus Nutzersicht würde es ja reichen einen Call zu haben, solange dieser Asynchron ausgeführt werden kann. Auf Serverseite ist das unterschiedlich, weil 1. beim Browserdownload (damals) keine Authentifizerungsinformationen mitgeschickt werden konnte und 2. die Erzeugung der Zipdatei länger dauern könnte (asynchron).